rtb/analisi_dei_requisiti_ver0.3.0.typ

```diff
@@ -0,0 +1,325 @@
+      - Modifiche alle sez. Attori e UC1: aggiunto sensore
+      - Aggiunto NiFi alle tecnologie utilizzate
+    ],
+    "0.1.0",
+    [25/11/2024],
+    "Davide Martinelli\nRiccardo Milan",
+    "Davide Marin\nDavide Picello",
+    [
+      - Struttura e introduzione del documento
+      - Descrizione del prodotto
+      - Primi casi d'uso
+    ],
+  ),
+  content: content,
+)
+
+
+#set heading(numbering: "1.1")
+
+= Introduzione
+== Scopo del documento
+
+Lo scopo principale del documento Analisi dei Requisiti è quello di fornire una descrizione dettagliata dei casi d'uso e dei requisiti del progetto _NearYou - Smart custom advertising platform_ dell'azienda proponente Sync Lab S.r.l., frutto di uno studio approfondito del capitolato d'appalto e degli incontri con l'azienda proponente stessa.
+
+Il documento risulta di fondamentale importanza in quanto costituisce una chiara visione delle funzionalità e i requisiti attesi riguardo al prodotto e facilita ai fornitori, ovvero il gruppo, nel processo di validazione del prodotto finale.
+
+
+== Scopo del prodotto
+
+Il prodotto _NearYou - Smart custom advertising platform_ è una piattaforma che sfrutta la #rifGlossario("GenAI") per la creazione di pubblicità personalizzate da mostrare a ciascun utente, sfruttando dati come la posizione in tempo reale, le informazioni personali e i dati di profilazione, in modo da migliorare l'esperienza dell'utente e aumentando contemporaneamente il #rifGlossario("ROI") e l'efficacia delle campagne pubblicitarie.
+
+== Glossario
+Per evitare eventuali ambiguità e incomprensioni sulla terminologia adottata nella documentazione redatta dal gruppo,
+viene fornito un Glossario.
+
+La prima occorrenza di un termine definito all'interno del Glossario presente all'interno di un documento viene
+sottolineato e seguito dalla lettera g posta ad apice (e.g. #rifGlossario("termine")).
+
+== Riferimenti
+=== Riferimenti normativi
+- Norme di Progetto
+- Regolamento del progetto didattico\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Dispense/PD1.pdf")
+- Capitolato C4 (Sync Lab)\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Progetto/C4.pdf")
+
+=== Riferimenti informativi
+- Glossario
+- T5 - Analisi dei Requisiti\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Dispense/T05.pdf")
+- P2 - I Diagrammi dei Casi d'Uso\ #formatLink(url: "https://www.math.unipd.it/~rcardin/swea/2022/Diagrammi%20Use%20Case.pdf")
+#pagebreak()
+
+= Descrizione del prodotto
+== Obiettivi del prodotto
+Il prodotto ha come obiettivo quello di fornire un servizio di pubblicità personalizzata integrato in una mappa, accessibile agli utenti di un mezzo a noleggio, che sfrutta la GenAI per la creazione di pubblicità mirate sui singoli utenti. Il gestore del servizio di noleggio avrà accesso ad una _dashboard_ che gli permetterà di monitorare in tempo reale la posizione degli utenti e la visualizzazione degli annunci pubblicitari.
```

Comment:
- [ ] va spiegato meglio che l'utente non vede la mappa, ma solo il messaggio apparire, è il gestore che vede la mappa.
- [ ] in questo paragrafo parlerei anche della profilazione dell'utente fondamentale per generare l'annuncio

Reply:
> * [ ]  va spiegato meglio che l'utente non vede la mappa, ma solo il messaggio apparire, è il gestore che vede la mappa.

Piccola distrazione mia. Procedo a correggere

Reply:
Ho modificato un po' il paragrafo. Dimmi cosa ne pensi (appena pusho ovviamente)

Reply:
(Mi rendo conto ora che era roba di martinelli, infatti mi sembrava strano)

Reply:
apposto